charts/kubewarden-controller/Chart.yaml

```diff
@@ -19,9 +19,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.0-rc5
+version: 1.6.0-rc6
```

Comment:
I think there's a bug. This should be `1.5.0-rc6`

Reply:
I think this is correct, we want to release `1.6.0`.

Reply:
No, you are right, this is the chart version, independent of the appVersion. It should indeed be `1.5.0-rc6`.

Reply:
solved in https://github.com/kubewarden/helm-charts/pull/220/commits/b45ff85d5ae520be5fa36187ac40b61c18e682b5.